packages/wdio-cucumber-framework/src/utils.ts

```diff
@@ -115,7 +108,7 @@ export function buildStepPayload(
 ) {
     return {
         uid: step.id,
-        title: createStepTitle(step),
+        title: isStepTypeHook(step) ? `hook-${step.hookId}` : step.text,
```

Comment:
I would set this to `Hook` here's an example of a hook in V6

```js
{
  uid: 'index.js187',
  title: 'Hook',
  parent: 'LoginPage: As a user I want to be able to login',
  argument: undefined,
  file: 'test/features/login.feature',
  tags: [ { type: 'Tag', location: [Object], name: '@happy-flow' } ],
  keyword: 'Hook',
  featureName: 'LoginPage',
  scenarioName: 'As a user I want to be able to login',
  type: 'hook'
}
```

and a Step in V6

```js
{
  uid: 'I load the Swag Labs demo website5',
  title: 'Given I load the Swag Labs demo website',
  parent: 'LoginPage: As a user I want to be able to login',
  argument: undefined,
  file: 'test/features/login.feature',
  tags: [ { type: 'Tag', location: [Object], name: '@happy-flow' } ],
  keyword: 'Given ',
  featureName: 'LoginPage',
  scenarioName: 'As a user I want to be able to login',
  type: 'step',
  state: 'pass',
  error: undefined,
  duration: 221,
  passed: true
}
```

Reply:
Okay, I will amend this.